src/sidebar/util/annotation-metadata.js

```diff
@@ -2,16 +2,20 @@
  * Utility functions for querying annotation metadata.
  */
 
+/** @typedef {import('../../types/api').Annotation} Annotation */
+/** @typedef {import('../../types/api').TextPositionSelector} TextPositionSelector */
+/** @typedef {import('../../types/api').TextQuoteSelector} TextQuoteSelector */
+
 /** Extract a URI, domain and title from the given domain model object.
  *
- * @param {object} annotation An annotation domain model object as received
- *   from the server-side API.
+ * @param {object} annotation
  * @returns {object} An object with three properties extracted from the model:
```

Comment:
It wasn't part of your changes, but I would recommend to remove the `@returns` line entirely for `documentMetadata`. It doesn't provide useful information to the reader beyond what is obvious from the code and the `{object}` type overrides the inferred return type with something less useful (`object` or `Object` to TS means "could be anything").

Reply:
I guess its good to point out that unless any return doc is very specific and asserts correctness, then we shouldn't add it in the first place.